x/evm/keeper/msg_server.go

```diff
@@ -184,7 +184,7 @@ func (s msgServer) VoteConfirmGatewayDeployment(c context.Context, req *types.Vo
 		return nil, fmt.Errorf("result of poll %s has wrong type, expected bool, got %T", req.PollKey.String(), poll.GetResult())
 	}
 
-	s.Logger(ctx).Info(fmt.Sprintf("%s gateway confirmation result is %t", chain.Name, confirmed.Value))
+	s.Logger(ctx).Info(fmt.Sprintf("%s gateway confirmation result is %t", chain.Name, confirmed.GetValue()))
```

Comment:
what's the diff between Value and GetValue?

Reply:
No idea, looks like we use `.Value` everywhere, switching back